src/lib.rs

```diff
@@ -989,6 +1036,19 @@ enum BuildInternal {
 
 impl Default for BuildInternal {
     fn default() -> Self {
-        BuildInternal::Never
+        Self::Never
+    }
+}
+
+impl FromStr for BuildInternal {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "auto" => Ok(Self::Auto),
+            "always" => Ok(Self::Always),
+            "never" => Ok(Self::Never),
+            v => Err(format!("Unknown value `{}`", v)),
```

Comment:
`strum` uses a simple enum for this:

```rust
pub enum ParseError {
    VariantNotFound,
}
```

It would seem better to have an actual error type here instead of just a string. If that error type even includes the variant that was not found then bonus points, but having an `Error` type is useful.